apps/condo/domains/document/access/Document.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Generated by `createschema document.Document 'organization:Relationship:Organization:CASCADE; property?:Relationship:Property:CASCADE; category:Relationship:DocumentCategory:CASCADE; file:File;'`
+ */
+
+
+const get = require('lodash/get')
+const isEmpty = require('lodash/isEmpty')
+const omit = require('lodash/omit')
+const uniq = require('lodash/uniq')
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+const { getById } = require('@open-condo/keystone/schema')
+
+const { queryOrganizationEmployeeFor } = require('@condo/domains/organization/utils/accessSchema')
+const { checkOrganizationPermission } = require('@condo/domains/organization/utils/accessSchema')
+
+
+const AVAILABLE_TO_UPDATE_DOCUMENT_FILE_FIELDS = ['dv', 'sender', 'deletedAt', 'category']
+
+async function canReadDocuments ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    if (user.isAdmin) return {}
+
+    return { organization: queryOrganizationEmployeeFor(user.id, 'canReadDocuments') }
```

Comment:
docs are not available for cc?

Reply:
It seems to me that this is not important in the MVP